scripts/highlight.sh

```diff
@@ -2,9 +2,9 @@
 
 ROOT_PATH=$(python -c "import os,sys;print(os.path.dirname(os.path.realpath(sys.argv[1])))" $0)
 ROOT_PATH=$(dirname $ROOT_PATH)
-npm --prefix $ROOT_PATH/_js/highlight install
 OLD_CWD=$(pwd)
 cd $ROOT_PATH/_js/highlight
 rm -rf build/ &> /dev/null
+npm install
```

Comment:
you can use `yarn` here if it's run on Netlify, or we can track the built version, since this will probably almost never change 🤔